api/qrcodegenerator.proto

```diff
@@ -90,3 +90,52 @@ service EndDeviceQRCodeGenerator {
     };
   };
 }
+
+// EndDeviceOnboardingData contains fields necessary for onboarding a LoRaWAN End Device.
+message EndDeviceOnboardingData {
+	bytes join_eui = 1 [(gogoproto.customtype) = "go.thethings.network/lorawan-stack/v3/pkg/types.EUI64"];
+  bytes dev_eui = 2 [(gogoproto.customtype) = "go.thethings.network/lorawan-stack/v3/pkg/types.EUI64"];
+  // claim_authentication_code ids referred to as;
+  // - Owner Token in LoRa Alliance TR005 specification.
+  // - Device Validation Code in the LoRa Alliance TR005 draft 2 and draft 3 specs.
+  string claim_authentication_code = 3;
+  bytes vendor_id = 4;
+  // model_id is referred to as VendorProfileID in LoRa Alliance TR005 specification.
+  bytes model_id = 5;
+  string checksum = 6;
+  string serial_number = 7;
+  string proprietary = 8;
```

Comment:
The checksum isn't a string. Also these fields are very specific to TR005 and are not strictly necessary for onboarding.

Did you consider returning an end device template instead of this onboarding data?

Reply:
Ah I see ok. Well no because I didn't know until now that `EndDeviceTemplate` exists. I'll update that here and re-request.